docs/usage/project.swift.mdx

```diff
@@ -814,6 +822,53 @@ Arguments contain commandline arguments passed on launch and Environment variabl
   ]}
 />
 
+### Archive Action
+
+<PropertiesTable
+  props={[
+    {
+      name: 'Configuration Name',
+      description:
+        'Indicates the build configuration to run the archive with.',
+      type: 'String',
+      optional: false,
+      default: '',
+    },
+    {
+      name: 'Reveal Archive in Organizer',
+      description:
+        "Indicates if the archive should revealed in Organizer on completion.",
```

Comment:
The previous description you used single quotes `'` and here you used double `"` is this intentional?